nixos/modules/services/mail/mailman.nix

```diff
@@ -253,14 +258,14 @@ in {
                 --arg archiver_key "$hyperkittyApiKey" \
                 --arg secret_key "$secretKey" \
                 >"$mailmanWebCfgTmp"
-            chown ${cfg.webUser} "$mailmanWebCfgTmp"
+            chown ${cfg.webUser}:mailman "$mailmanWebCfgTmp"
```

Comment:
The config contains secrets. What reason would there be for another service to want to read this file?

Reply:
Oh you're right - this file should not be shared. I'll revert this part.